src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -40,7 +40,7 @@ public void BuildBasicApplicationRelease ()
 		[Test]
 		public void BuildHasNoWarnings ([Values (true, false)] bool isRelease)
 		{
-			var proj = new XamarinAndroidApplicationProject {
+			var proj = new XamarinFormsAndroidApplicationProject {
```

Comment:
Should we have two tests one for XF and one for normal XA? I ask because the templates will be different and will have different resources.